java/src/jmri/ProgrammingMode.java

```diff
@@ -1,6 +1,8 @@
 /* ProgrammingMode.java */
 package jmri;
 
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
I thought we wanted to use ```javax.annotations.Nonnull``` (there was an older email that mentioned migrating from findbugs annotations to javax annotations on jmri-developers).